src/apple/apple-wizard.ts

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-lines */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as templates from './templates';
+import * as xcManager from './xcode-manager';
+import * as codeTools from './code-tools';
+import * as bash from '../utils/bash';
+import { WizardOptions } from '../utils/types';
+
+const xcode = require('xcode');
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+import {
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  askToInstallSentryCLI,
+  SentryProjectData,
+  printWelcome,
+} from '../utils/clack-utils';
+
+export async function runAppleWizard(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Apple Wizard',
+    promoCode: options.promoCode,
+  });
+
+
+  if (!bash.hasSentryCLI()) {
+    if (!(await askToInstallSentryCLI())) {
+      clack.log.warn("Without Sentry-cli, you won't be able to upload debug symbols to Sentry. You can install it later by following the instructions at https://docs.sentry.io/cli/");
+    } else {
+      await bash.installSentryCLI();
+    }
+  }
+
+  const projectDir = process.cwd();
+  const xcodeProjFile = findFilesWithExtension(projectDir, ".xcodeproj")[0];
+
+  if (!xcodeProjFile) {
+    clack.log.error('No xcode project found. Please run this command from the root of your project.');
+    return;
+  }
+
+  const pbxproj = path.join(projectDir, xcodeProjFile, "project.pbxproj");
+  if (!fs.existsSync(pbxproj)) {
+    clack.log.error(`No pbxproj found at ${pbxproj}`);
+    return;
+  }
+
+  const { project, apiKey } = await getSentryProjectAndApiKey(options.promoCode, options.url);
+
+  xcManager.updateXcodeProject(pbxproj, project, apiKey, true, true);
+
+  const codeAdded = codeTools.addCodeSnippetToProject(projectDir, project.keys[0].dsn.public);
+  if (!codeAdded) {
+    clack.log.warn('Sentry dependency was added to your project, but could not add Sentry code snippet to it. Please add it manually by following this: https://docs.sentry.io/platforms/apple/guides/ios/#configure');
```

Comment:
Maybe a bit cleaner:
```suggestion
    clack.log.warn('Added the Sentry dependency to your project but could not add the Sentry code snippet. Please add the code snipped manually by following the docs: https://docs.sentry.io/platforms/apple/guides/ios/#configure');
```